Sources/Subs-Graphics.php

```diff
@@ -1062,43 +1062,43 @@ function showCodeImage($code)
 	// Add some noise to the background?
 	if ($noiseType != 'none')
 	{
-		for ($i = mt_rand(0, 2); $i < $max_height; $i += mt_rand(1, 2))
-			for ($j = mt_rand(0, 10); $j < $total_width; $j += mt_rand(1, 10))
-				imagesetpixel($code_image, $j, $i, mt_rand(0, 1) ? $fg_color : $randomness_color);
+		for ($i = random_int(0, 2); $i < $max_height; $i += random_int(1, 2))
+			for ($j = random_int(0, 10); $j < $total_width; $j += random_int(1, 10))
+				imagesetpixel($code_image, $j, $i, random_int(0, 1) ? $fg_color : $randomness_color);
```

Comment:
This change is not needed.This change is not needed.This change is not needed.This change is not needed.This change is not needed.

Reply:
Getting a little trigger-happy on the copy & paste, eh? 😆

Reply:
/me giggles

There's a pattern here...